sections/shared/Layout/SideNav/PriceItem.tsx

```diff
@@ -93,32 +98,9 @@ const PriceInfo = styled(FlexDivCentered)`
 	justify-content: space-between;
 `;
 
-const PercentChange = styled.div<{ trendLinePositive: boolean }>`
+const PercentChange = styled.div`
 	font-size: 10px;
 	font-family: ${(props) => props.theme.fonts.interBold};
-	color: ${(props) =>
```

Comment:
see comment above about reintroducing the correct colors

```
green is up
pink is down
```

Reply:
@andytcf Isnt that already handled in `ChangePercent.tsx`?
![image](https://user-images.githubusercontent.com/52366477/116535598-37604d00-a8e4-11eb-9902-4f16a72dfaf1.png)

The only difference between the percentage design in the AssetTable from the design in the side nav is the font-family and font-size.

I usually use css modules, havent worked with styled-components before so maybe i dont understand the how things styles are inherited to child doms. 


You mean something like this:

```
<PercentChange isPositive={historicalRates.data?.change >= 0}>
      <ChangePercent className="percent" value={historicalRates.data?.change} />
</PercentChange>

const PercentChange = styled.div<{ isPositive: boolean }>`
	font-size: 10px;
	font-family: ${(props) => props.theme.fonts.interBold};
	color: ${(props) => (props.isPositive ? props.theme.colors.green : props.theme.colors.pink)};
`;
```

However it feels redundant to set color here when it is already handled by the `ChangePercent` component?